db/migrate/20220919141351_change_consent_question_in_study_questions.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeConsentQuestionInStudyQuestions < ActiveRecord::Migration[5.2]
+  def change
+    @stq = StudyTypeQuestion.find(13)
```

Comment:
Finding by ID can be dangerous.   I think this might be better suited as a post deploy manual fix but maybe others can chime in

Reply:
I just updated the migration, if that helps.  Otherwise, we can go with your idea, if necessary.  I just feel like it does need to be a migration just to make sure that anyone who pulls the code has the most current things in the database since these records are programmatically significant instead of arbitrary data.

Reply:
@Ito-Eta This is great work, but I think I agree with @amcates here in that this should be a rake task instead of a migration. We should only be using migrations for schema changes (the exception here is if you need to move data around as the _result_ of a schema change). If you put a note in the release marker your rake task will get run when production is updated. Me and Sherly have a couple of them in there.

Reply:
Okay.  Updated to rake task, the migration was deleted, and a note was added to the release marker.